packages/date-picker/src/DatePicker/DatePickerMenu/DatePickerMenuHeader/utils/isChevronDisabled/index.ts

```diff
@@ -0,0 +1,23 @@
+import { isSameUTCMonth } from '../../../../../shared/utils';
+
+/**
+ * Checks if chevron should be disabled
+ *
+ * @param direction left or right chevron
+ * @param day1 the full date of the current month shown in the menu (month)
+ * @param day2 the full date that current menu date is compared against (min/max)
+ * @returns
+ */
+export const isChevronDisabled = (
+  direction: 'left' | 'right',
+  day1: Date,
+  day2: Date,
+): boolean => {
+  if (!day1 || !day2) return false;
+
+  if (direction === 'right') {
+    return day1 >= day2 || isSameUTCMonth(day1, day2);
```

Comment:
I'm not sure comparing dates like this is the most reliable. Though, if your tests are passing, it should be fine!